utils/config.go

```diff
@@ -24,6 +24,7 @@ type ConfigStructure struct {
 	SkipContentsPublishing bool                        `json:"skipContentsPublishing"`
 	S3PublishRoots         map[string]S3PublishRoot    `json:"S3PublishEndpoints"`
 	SwiftPublishRoots      map[string]SwiftPublishRoot `json:"SwiftPublishEndpoints"`
+	APIHookCmd             string                      `json:"APIHookCmd"`
```

Comment:
This breaks system tests, so simple fix is required to make tests happy again...